code/modules/research/xenobiology/crossbreeding/mutative.dm

```diff
@@ -0,0 +1,150 @@
+//All mutative extracts result in items so let's just put them in a single file for the ease of readability.
+/obj/item/slimecross/mutative
+	icon_state = "mutative"
+	name = "mutated"
+	effect = "mutative"
+
+/obj/item/slimecross/mutative/darkpurple
+	colour = "dark purple"
+	effect_desc = "Fully stocks up any pacman type generator with fuel."
+
+/obj/item/slimecross/mutative/darkblue
+	colour = "dark blue"
+	effect_desc = "Turns 50u of water into single stack of snow."
+
+/obj/item/slimecross/mutative/darkblue/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!target.reagents)
+		return
+	var/datum/reagents/reggies = target.reagents
+	var/datum/reagent/water = reggies.has_reagent(/datum/reagent/water)
+	if(!water)
+		return
+	var/sheet_num = FLOOR(water.volume / 50,1)
+	if(!sheet_num)
+		return
+	playsound(src, 'sound/effects/bubbles.ogg', 50, TRUE)
+	reggies.remove_reagent(/datum/reagent/water,sheet_num*50)
+	var/locc = drop_location()
+	for(var/num in 1 to sheet_num)
+		new /obj/item/stack/sheet/mineral/snow(locc)
+
+/obj/item/slimecross/mutative/green
+	colour = "green"
+	effect_desc = "Single use core, use it in your hand to become immune to damage slow down and soft crit, but your blood becomes acid and if you die you are gibbed."
+
+/obj/item/slimecross/mutative/green/attack_self(mob/user)
+	. = ..()
+	if(do_after(user,10 SECONDS,TRUE,user))
+		playsound(src, 'sound/effects/bubbles.ogg', 50, TRUE)
+		to_chat(user, "<span class='warning'>You feel your whole body shift...</span>")
+		user.emote("scream")
+		ADD_TRAIT(user,TRAIT_DISFIGURED,type)
+		ADD_TRAIT(user,TRAIT_GIB_DEATH,type)
+		ADD_TRAIT(user,TRAIT_IGNOREDAMAGESLOWDOWN,type)
+		ADD_TRAIT(user,TRAIT_NOSOFTCRIT,type)
+		if(!istype(user,/mob/living/carbon/human))
+			return
+		var/mob/living/carbon/human/human_user = user
+		human_user.dna.species.exotic_blood = /datum/reagent/toxin/acid/nitracid
+		human_user.AdjustParalyzed(5 SECONDS)
+
+/obj/item/slimecross/mutative/oil
+	colour = "oil"
+	effect_desc = "Provides long-lasting mood buff."
+
+/obj/item/slimecross/mutative/oil/attack_self(mob/user)
+	. = ..()
+	playsound(src, 'sound/effects/bubbles.ogg', 50, TRUE)
+	SEND_SIGNAL(user,COMSIG_ADD_MOOD_EVENT,"oil_buff",/datum/mood_event/oil_buff)
+	qdel(src)
+
+/obj/item/slimecross/mutative/item_spawner
+	effect_desc = "Use in hand to recieve the corresponding item."
+	///Item to spawn
+	var/item_spawner
+
+/obj/item/slimecross/mutative/item_spawner/item_spawner/attack_self(mob/user)
+	. = ..()
+	if(item_spawner)
+		playsound(src, 'sound/effects/bubbles.ogg', 50, TRUE)
+		new item_spawner(drop_location())
+		qdel(src)
+
+/obj/item/slimecross/mutative/item_spawner/grey
+	item_spawner = /obj/item/slimeball
+	colour = "grey"
+
+/obj/item/slimecross/mutative/item_spawner/orange
+	item_spawner = /obj/item/spear/slime
```

Comment:
wait hold up

when I read the code for the slime spear, I assumed that you'd be able to make them with an ability or item that could summon one once every 30 seconds or something. this is waaaaaaaaaay too weak of an item for someone sane/well-informed to blow 11 slime cores on. a NORMAL SPEAR is better than a slime spear (especially since the main value of spears as a throwing weapon are their ability to embed in people), and you can craft normal spears as a goddamn PRISONER.

again, you're spending 11 SLIME CORES on a SINGLE, WORSE-THAN-NORMAL SPEAR.

either make this extract reusable (with a cooldown of, say, 30 seconds), or make it summon a slime fire axe or something.